backend/src/main/java/com/example/backend/controller/ApiController.java

```diff
@@ -0,0 +1,81 @@
+package com.example.backend.controller;
+
+import com.example.backend.Repositories.*;
+import com.example.backend.model.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Sort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
+public class ApiController {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private BankAccountRepository bankAccountRepository;
+
+    @Autowired
+    BalanceHistoryRepository balanceHistoryRepository;
+
+    @Autowired
+    GoalRepository goalRepository;
+
+    @Autowired
+    GoalElementRepository goalElementRepository;
+
+
+    @GetMapping("/")
+    public String save(@RequestParam(value = "name") String name){
+
+        User user = new User(name,"hehehaslo","test@gmail.com");
+
+        userRepository.save(user);
+
+        BankAccount bankAccount1 = new BankAccount("ING",1000.43, user);
+        BankAccount bankAccount2 = new BankAccount("Mbank", 1200.99, user);
+
+        bankAccountRepository.save(bankAccount1);
+        bankAccountRepository.save(bankAccount2);
+
+        balanceHistoryRepository.save(new BalanceHistory(bankAccount1, java.sql.Date.valueOf("2020-11-15"), java.sql.Date.valueOf("2022-11-15"),30,200.00,"fajna platnosc", ActionType.Przychód, "Netflix"));
+        balanceHistoryRepository.save(new BalanceHistory(bankAccount1, java.sql.Date.valueOf("2021-11-15"), java.sql.Date.valueOf("2023-11-15"),30,230.00,"xdxdxxxdxdx", ActionType.Wydatek, "brak"));
+
+        Goal goal = new Goal("remont",user);
+
+        goalRepository.save(goal);
+
+        GoalElement goalElement1 = new GoalElement("podloga",2000,goal);
+        GoalElement goalElement2 = new GoalElement("szafa",700,goal);
+
+        goalElementRepository.save(goalElement1);
+        goalElementRepository.save(goalElement2);
+
+        return "dodano_wydatek";
+    }
+
+    @GetMapping("/addIncome")
+    public String sendIncomeExpensesForm(Model model){
+        model.addAttribute("balanceHistory", new BalanceHistory());
+        return "new_income_expense";
+    }
+
+    @PostMapping("/addIncome")
+    public ResponseEntity processIncomeExpenseForm(@ModelAttribute BalanceHistory balanceHistory){
+        List<User> users = userRepository.findAll(Sort.by(Sort.Direction.ASC, "id"));
+        User user = users.get(0);
```

Comment:
to co napisałem w userRepository, fajnie by było żeby od razu był zwracany pierwszy użytkownik a nie lista z której pobieramy.